common/source/docs/common-remoteid.rst

```diff
@@ -0,0 +1,20 @@
+.. _common-remoteid:
+
+[copywiki destination="copter,plane,rover"]
+
+========================
+Remote ID (aka Drone ID)
+========================
+
+.. image:: ../../../images/remoteid-title.png
+    :target: http://www.eams-robo.co.jp/remoteid.html
+    :width: 400px
+
+Remote IDs are becoming a legal requirement in some countries.  Below is a list of devices compatible with ArduPilot
+
+- `Aerobits idME and idME+ <https://www.aerobits.pl/product/idme/>`__
+- `BlueMark DroneBeacon MAVLink (EU, US, Japan) <https://dronescout.co/dronebeacon-mavlink-remote-id-transponder/>`__
+- `Dronetag Mini <https://dronetag.cz/en/products/mini/>`__
+- `EAMS Robotics remote id (Japan) <http://www.eams-robo.co.jp/remoteid.html>`__
```

Comment:
https://www.aerobits.pl/product/idme-remoteid/

Reply:
ok, separated the two aerobits products.